### ISSUE TITLE
BeerCSS with NextJs

### ISSUE THREAD

**Issue by aimproxy** [beercss/beercss#111]
![Screenshot 2022-12-29 at 18-41-24 Screenshot](https://user-images.githubusercontent.com/47716563/209995482-9cc01dbe-f6fd-4e7a-aae0-e83f644e8a65.png)

## How I get way with this?
```tsx
// Under _app.tsx

import "beercss/src/cdn/beer.css";
import "beercss/src/cdn/beer.ts";
import "material-dynamic-colors";
```

## What I get in response? Yet another error!
![Screenshot from 2022-12-29 18-46-20](https://user-images.githubusercontent.com/47716563/209995925-6154dd46-559f-4ee0-9ad5-a7714c3ecf28.png)

## So I rollback and try this:
```tsx
// Under _document.tsx
export default function Document() {
  return (
    <Html>
      <Head>
        <link href="https://cdn.jsdelivr.net/npm/beercss@2.3.0/dist/cdn/beer.min.css" rel="stylesheet"/>
        {/* eslint-disable-next-line @next/next/no-sync-scripts */}
        <script src="https://cdn.jsdelivr.net/npm/beercss@2.3.0/dist/cdn/beer.min.js" type="text/javascript"/>
        {/* eslint-disable-next-line @next/next/no-sync-scripts */}
        <script
          src="https://cdn.jsdelivr.net/npm/material-dynamic-colors@0.0.10/dist/cdn/material-dynamic-colors.min.js"
          type="text/javascript"/>
      </Head>
      <body>
      <Main/>
      <NextScript/>
      </body>
    </Html>
  )
}
```

## And someone it loads:
![Screenshot 2022-12-29 at 18-57-59 Screenshot](https://user-images.githubusercontent.com/47716563/209996959-6a7c8c5a-4876-490e-8c78-7e6f9a73ebfe.png)

**Comment by leonardorafael** [comment]
Will be available in 3.0.0 version.

**Comment by leonardorafael** [comment]
We added more import options for more uses cases https://github.com/beercss/beercss/blob/main/docs/INDEX.md#npm
